liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -164,10 +186,21 @@ void SourceReferenceFormatter::printSourceLocation(SourceReference const& _ref)
 void SourceReferenceFormatter::printExceptionInformation(SourceReferenceExtractor::Message const& _msg)
 {
 	// exception header line
-	errorColored() << _msg.severity;
-	if (m_withErrorIds && _msg.errorId.has_value())
-		errorColored() << " (" << _msg.errorId.value().error << ")";
-	messageColored() << ": " << _msg.primary.message << '\n';
+	if (_msg.errorId.has_value())
+	{
+        switch(stringToSeverity(_msg.severity))
```

Comment:
The function returns `optional`. You must use `has_value()` to make sure it's not `nullopt` first.

Reply:
If I'm not wrong I've checked it at line 182 of `liblangutil/SourceReferenceFormatter.cpp` ?